workflows/genomeassembly.nf

```diff
@@ -46,9 +37,7 @@ include { INPUT_CHECK } from '../subworkflows/local/input_check'
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/modules/fastqc/main'
-include { MULTIQC                     } from '../modules/nf-core/modules/multiqc/main'
-include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'
```

Comment:
you want to keep `CUSTOM_DUMPSOFTWAREVERSIONS` to combine all your `versions.yml` files into one.